src/proxy-ui-api/src/main/java/org/niis/xroad/securityserver/restapi/openapi/SecurityServersApiController.java

```diff
@@ -73,13 +75,13 @@
 
     @Override
     @PreAuthorize("hasAuthority('VIEW_SECURITY_SERVERS')")
-    public ResponseEntity<List<SecurityServer>> getSecurityServers(Boolean currentServer) {
+    public ResponseEntity<Set<SecurityServer>> getSecurityServers(Boolean currentServer) {
         boolean getOnlyCurrentServer = Boolean.TRUE.equals(currentServer);
-        List<SecurityServer> securityServers = null;
+        Set<SecurityServer> securityServers = null;
         if (getOnlyCurrentServer) {
             SecurityServerId currentSecurityServerId = serverConfService.getSecurityServerId();
             SecurityServer currentSecurityServer = securityServerConverter.convert(currentSecurityServerId);
-            securityServers = Collections.singletonList(currentSecurityServer);
+            securityServers = new HashSet<>(Collections.singletonList(currentSecurityServer));
```

Comment:
I have to use Set. 
Because generated interface **SecurityServersApi** has a method getSecurityServers() with response type `ResponseEntity<Set<SecurityServer>> `. I can try to change OpenApiGenerator configuration to change response from Set to List?

Reply:
You can simplify ` new HashSet<>(Collections.singletonList(currentSecurityServer))` to `Collections.singleton(currentSecurityServer)` which is a Set.

See https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#singleton-T-

Reply:
fixed in https://github.com/nordic-institute/X-Road/pull/1042/commits/c7d8fa152ba6d424760c35a3067ce80f4b687612